Pod/Classes/Logging/Log.swift

```diff
@@ -49,11 +53,38 @@ open class Log {
         }
     }
 
+    public static private(set) var standard = Log("", logLevel: .off)
+
+    /// Static instance used for helper methods.
+    open class var instance: Log {
+        return standard
+    }
+
+    /// Subsystem of the OSLog message when running on iOS 14 or later.
+    open var subsystem: String {
+        let bundleName = Bundle.main.object(forInfoDictionaryKey: "CFBundleDisplayName") as? String
+        return (bundleName ?? "com.rightpoint.swiftilities") + ".log"
```

Comment:
>>>subsystem
A string that identifies the app subsystem in which to record messages. Specify this string using reverse DNS notation — for example, com.yourcompany.subsystem_name. The system uses this information to categorize and filter related log messages, and group related logging settings.